scripts/brigde/src/utils/wallets.ts

```diff
@@ -3,7 +3,7 @@ import type { WalletUnlocked } from 'fuels';
 import { Provider, Wallet } from 'fuels';
 
 export const ethProvider = new ethers.providers.JsonRpcProvider(
-  'http://localhost:9545'
+  'http://localhost:8545'
```

Comment:
I've seen the 8545 port a couple of times.  Is it worth it to put it in a variable?

Reply:
it's only inside `scripts` package, which will be removed when withdraw gets implemented

Reply:
for network connection we use `foundry` variable from wagmi that already contains URL with correct 8545 port